Docs for https://github.com/mautic/mautic/pull/16123 (#714)
mautic/user-documentation: docs/configuration/command_line_interface.rst

     - 
   * - ``social:monitor:twitter:mentions``
     - Searches for mentioned tweets
     -
   * - ``mautic:forms:delete-results-table``

>      - Deletes orphan form results tables for Forms that have been deleted. Use this to clean up database tables that were left behind when Forms were removed.

Reviewer: The word 'Form' should be capitalized and we need to use active instead of passive voice.

```suggestion
     - Deletes orphan results tables for deleted Forms.
```

     -
   * - ``mautic:forms:delete-orphan-form-submission-records-from-form-results-table``

>      - Deletes orphan records from form results tables where the associated Form submission has been deleted. Use this to clean up stale data from Form results tables.

Reviewer: The word 'Form' should be capitalized and we need to use active instead of passive voice.

```suggestion
     - Deletes orphan records from results tables for deleted Form submissions.
```
